src/backend/core/cms/views.py

```diff
@@ -0,0 +1,190 @@
+from rest_framework.status import HTTP_201_CREATED, HTTP_400_BAD_REQUEST
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.permissions import AllowAny
+from rest_framework.generics import RetrieveUpdateDestroyAPIView
+
+# from django.contrib.auth.models import User
+from .models import (
+    PageCategoryType,
+    PageCategory,
+    Page,
+    PageFrontend,
+    PageCMS,
+)
+
+from .serializers import (
+    PagePolymorphicDashboardSerializer,
+    PageCMSDashboardSerializer,
+    PageFrontendDashboardSerializer,
+    PageCategoryDashboardSerializer,
+    PageCateogryTypeDashboardSerializer,
+)
+
+
+class PageDashboardView(APIView):
+    """
+    View for adding new page (CMS or Frontend) and listing all pages
+    """
+
+    permission_classes = (AllowAny,)
+    authentication_classes = []
+
+    def get(self, request):
```

Comment:
I think we can use [ListCreateAPIView](https://www.django-rest-framework.org/api-guide/generic-views/#examples) here, can't we?

Reply:
Not 100% about it, when we will be dealing with translated content we might run into issues since we won't be able to pass `request` object into a serializer. I'd leave it like this and we will see.